dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/EncodingValidator.java

```diff
@@ -1,39 +0,0 @@
-/*******************************************************************************
```

Comment:
Instance of references to this class needs to be updated/removed. One such place would be OASValidator.visitEncoding method. There may be others.

Reply:
I've updated those, and did a gradle build to make sure everything compiles